### ISSUE TITLE
Parameters of type [array] not showing expected behavior.

### ISSUE THREAD

**Issue by Schandlich** [domaindrivendev/Swashbuckle#363]
If I were to have a controller method like so:

    public async Task<IHttpActionResult> Get([FromUri]string[] tags, [FromUri]int? skip = 0, [FromUri]int? top = 1000)

I would expect to have swagger ui render it like it is in the "pets" demo. http://petstore.swagger.io/#!/pet/findPetsByTags
![untitled](https://cloud.githubusercontent.com/assets/3946479/7870878/f617ebb6-0540-11e5-9f62-c7bc5d1bdc6d.png)

Instead, what I get is:
![untitled2](https://cloud.githubusercontent.com/assets/3946479/7870910/348310ce-0541-11e5-90fc-7760ef5309ac.png)

It appears that the json getting generated is the same:

Mine:
```json
{
   "name":"tags",
   "in":"query",
   "required":true,
   "type":"array",
   "items":{
      "type":"string"
   },
   "collectionFormat":"multi"
}
```

Demo:
```json
{
   "name":"tags",
   "in":"query",
   "description":"Tags to filter by",
   "required":true,
   "type":"array",
   "items":{
      "type":"string"
   },
   "collectionFormat":"multi"
}
```

Does anyone have any suggestions?

**Comment by domaindrivendev** [comment]
This is due to SB bundling a slightly older version of swagger-ui. Will be updated with the next release

**Comment by domaindrivendev** [comment]
Should be resolved in latest release - 5.2.0

**Comment by Schandlich** [comment]
Thank you!
